motor_subscriber:
  ros__parameters:
    arduino_port: "/dev/ttyACM0"
    baud_rate: 9600